pdfarranger/pdfarranger.py

```diff
@@ -533,9 +533,11 @@ def __create_filters(file_type_list):
             filter_list.append(f_img)
         return filter_list
 
-    def set_title(self, title):
+    def set_title(self, title, subtitle = None):
         if Handy:
             self.uiXML.get_object('header_bar').set_title(title)
+            if subtitle is not None:
```

Comment:
It might be slightly better to set this always, so that a previously set subtitle can be reset. (I don't know if it should be set to None or something else, someone would need to look that up.)

Reply:
I think that is even necessary, what happens if you start empty, load a file and Ctrl+Z back? I think the subtitle might be left behind in this case. Could you check that?

Reply:
You're right, looks like `None` is an acceptable argument anyway: https://lazka.github.io/pgi-docs/#Handy-1/classes/HeaderBar.html#Handy.HeaderBar.set_subtitle

Reply:
> I think that is even necessary, what happens if you start empty, load a file and Ctrl+Z back? I think the subtitle might be left behind in this case. Could you check that?

I'm not sure I understand this. For me, the subtitle doesn't appear until a file is saved, as there is no path to show until then. Is that the desired behavior?

Reply:
You're right, sorry. It is now fixed and future proof anyway, thanks!